docs/autonomy.md

```diff
@@ -37,6 +37,12 @@ Options:
   --open-aea-dir PATH             Path to open-aea repo (Use with dev mode)
   --open-autonomy-dir PATH        Path to open-autonomy repo (Use with dev
                                   mode)
+  --aev                           Apply environment variable when loading
+                                  service config.
+  --use-hardhat                   Include a hardhat node in the deployment
+                                  setup.
+  --use-acn                       Include an ACN node in the deployment setup.
+  --image-version TEXT            Define runtime image version.
```

Comment:
@jmoreira-valory I notice below we use the oracle for demo purposes. Please replace below with the hello world service so that we can keep the oracle in one place only.

Reply:
I've created the ticket for jose